packages/compartment-mapper/src/import-hook.js

```diff
@@ -189,6 +190,21 @@ const findRedirect = ({
         };
       }
 
+      if (someCompartmentDescriptor.path && compartmentDescriptor.policy) {
```

Comment:
We should make sure that if policy is truthy and path is not we don't end up allowing something by default. 
I'd limit the if statement to just policy and react to path missing inside the if block (with a `continue` statement if that's appropriate, or with an error. Not sure which one that would be right now.